index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>PORTFOLIO: SETUP AND MOBILE VERSION SKELETON</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="container">
+        <header>
+            <div id="toolbar">
+                <nav class="left">
+                    <img src="Time.png" alt="">
+                </nav>
+                <nav class="right">
+                    <img src="Cellular.png" alt="Cellular">
+                    <img src="Wifi.png" alt="Wifi">
+                    <img src="Battery.png" alt="battery">
+                </nav>
+                <nav>
+                    <div class="toolbar1">
+                        <img src="Logo.png" alt="Logo">
+                    </div>
+                    <div class="toolbar2">
+                        <img src="Union.png" alt="Menu">
+                    </div>
+                </nav> 
+            </div>
+        </header>
+        <main class="name">
+            <p>I'm Kelvin Glad to see you!</p>
+        </main>
+        <div class="text">
+            <p>I'm a software developer! I can help you build a product,
+                feature or website. Look through someof my work and experience!
+                If you like what you see and have a project you need coded,
+                don't hesitate to contact me.
+            </p>
+        </div>
+        <nav id="lets-connect">
+            <h5>LET'S CONNECT</h5>
+        </nav>
+        <div class="icons">
+            <img src="icons.png" alt="Social Media Icons">
+        </div>
```

Comment:
- [ ] Please put these highlighted lines of code in a `section` tag. That will make it easier to read because they belong to the headline of your portfolio.